### PR TITLE
Generates n blobs.

### PR DESCRIPTION
**Pull request by bonafidehan**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -278,9 +278,41 @@ export function genHLines(width, height, options, override) {
...
+	for (let i = 2; i < opt.numControls; i ++) {
+		path += "S " + data[i].ctrl + ", " + data[i].point + " "
+	}
+	path += "S " + data[0].ctrl + ", " + data[0].point
+
+	return({path: path, data: data})
+}
+
 export function genBlob(size, options) {
+	genHBlobs(size, options)
```
Comment (artt) [inline src/index.js]: Need to return the 0th element since `genHBlobs` now returns an array.
```suggestion
	return genHBlobs(size, options)[0]
```

**Review by artt — CHANGES_REQUESTED** [review]
Thanks! Looks like you need to add a `return` to make it backward compatible. Function names look not as I initially intended (`H` in `genHLine` came from "horizontal lines" haha) but should be fine for now. Will clean up the code in the next major version (if any).

**Comment by bonafidehan** [comment]
Oops thanks for catching that!

**Review by artt — APPROVED** [review]
Looks good!

**Pull request merged**